src/test/java/net/snowflake/client/jdbc/SnowflakeDriverLatestIT.java

```diff
@@ -1108,4 +1115,76 @@ public void testPutS3RegionalUrl() throws Throwable {
       }
     }
   }
+
+  /**
+   * Test the streaming ingest client name and client key should be part of the file metadata in S3
+   * and Azure
```

Comment:
What about GCP or is that coming in a future PR?

Reply:
NVM I see your comment - yeah I ran into that server side, there was some black magic I had to invoke to set things up correctly.

Reply:
hmmm, i need black magic as well, could you share it :)